src/DssSpell.sol

```diff
@@ -65,312 +72,282 @@ contract DssSpellAction is DssAction {
-        // Poll: https://vote.makerdao.com/polling/QmYHUDVA#poll-detail
+        // Add RWA013-A to the LineMom
+        LineMomLike(LINE_MOM).addIlk("RWA013-A");
+
+        // Add RWA015-A to the LineMom
+        LineMomLike(LINE_MOM).addIlk("RWA015-A");
+
+        // ---------- Add emergency spells to the chainlog ----------
+        // Forum: https://forum.sky.money/t/standby-spells-for-sky-emergency-response/25594
+        // Poll: https://vote.makerdao.com/polling/QmRfTQ5t
+
+        // Contract: SingleClipBreakerFactory - Key: EMSP_CLIP_BREAKER_FAB - Value: 0xd7321d0919573a33f9147fD2579a48F60237564A
+        DssExecLib.setChangelogAddress("EMSP_CLIP_BREAKER_FAB", 0xd7321d0919573a33f9147fD2579a48F60237564A);
+
+        // Contract: SingleDdmDisableFactory - Key: EMSP_DDM_DISABLE_FAB - Value: 0x8BA0f6C4009Ea915706e1bCfB1d879E34587dC69
+        DssExecLib.setChangelogAddress("EMSP_DDM_DISABLE_FAB", 0x8BA0f6C4009Ea915706e1bCfB1d879E34587dC69);
+
+        // Contract: SingleLineWipeFactory - Key: EMSP_LINE_WIPE_FAB - Value: 0xA649730fA92695096b7C49DBae682995F8906684
+        DssExecLib.setChangelogAddress("EMSP_LINE_WIPE_FAB", 0xA649730fA92695096b7C49DBae682995F8906684);
+
+        // Contract: SingleOsmStopFactory - Key: EMSP_OSM_STOP_FAB - Value: 0x83211c74131bA2B3de7538f588f1c2f309e81eF0
+        DssExecLib.setChangelogAddress("EMSP_OSM_STOP_FAB", 0x83211c74131bA2B3de7538f588f1c2f309e81eF0);
 
-        // Transfer 4,000,000 DAI to the Pause Proxy from the Surplus Buffer
-        DssExecLib.sendPaymentFromSurplusBuffer(MCD_PAUSE_PROXY, 4_000_000);
+        // Contract: MultiClipBreakerSpell - Key: EMSP_GLOBAL_CLIP_BREAKER - Value: 0x828824dBC62Fba126C76E0Abe79AE28E5393C2cb
+        DssExecLib.setChangelogAddress("EMSP_GLOBAL_CLIP_BREAKER", 0x828824dBC62Fba126C76E0Abe79AE28E5393C2cb);
 
-        // Note: we have to approve DAI_USDS contract to convert DAI into USDS
-        DAI.approve(DAI_USDS, 4_000_000 * WAD);
+        // Contract: MultiLineWipeSpell - Key: EMSP_GLOBAL_LINE_WIPE - Value: 0x4B5f856B59448304585C2AA009802A16946DDb0f
+        DssExecLib.setChangelogAddress("EMSP_GLOBAL_LINE_WIPE", 0x4B5f856B59448304585C2AA009802A16946DDb0f);
 
-        // Convert 4,000,000 DAI to USDS using DAI_USDS
-        // Note: this is done by the next line of code
+        // Contract: MultiOsmStopSpell - Key: EMSP_GLOBAL_OSM_STOP - Value: 0x3021dEdB0bC677F43A23Fcd1dE91A07e5195BaE8
+        DssExecLib.setChangelogAddress("EMSP_GLOBAL_OSM_STOP", 0x3021dEdB0bC677F43A23Fcd1dE91A07e5195BaE8);
 
-        // Transfer 4,000,000 USDS from PauseProxy to 0xD8507ef0A59f37d15B5D7b630FA6EEa40CE4AFdD
-        DaiUsdsLike(DAI_USDS).daiToUsds(LIQUIDITY_BOOTSTRAPPING, 4_000_000 * WAD);
+        // Note: Bump chainlog patch version as new keys are being added
+        DssExecLib.setChangelogVersion("1.19.4");
+
+        // ---------- Launch Project Funding ----------
```

Comment:
Please add the [missing Authority URL](https://docs.google.com/spreadsheets/d/1w_z5WpqxzwreCcaveB2Ye1PP5B8QAHDglzyxKHG3CHw/edit?gid=485154570#gid=485154570&range=H85) to the Atlas to this item

Reply:
Done